recipes/warp/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "Warp" %}
+{% set version = "2.0.0dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/alisterburt/pandas/releases/download/v1.2.0.dev0/warp-main.tar.gz  # temporary
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  sha256: b76a3070468084ddf9c0760a3126c01037dcc311e6b59a298a89b7069b77b668
+
+build:
+  skip: true  # [win or osx or cuda_compiler_version == "None"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - dotnet ==6.0
+    - pytorch ==2.1.0
+    - libtorch ==2.1.0
+    - fftw
+    - libtiff
+    - zlib
+    - cudnn
+    - python
+  host:
+    - dotnet ==6.0
+    - pytorch ==2.1.0
+    - libtorch ==2.1.0
+    - fftw
+    - libtiff
+    - zlib
+    - torchvision
+    - cudnn
+  run:
+    - dotnet ==6.0
+    - pytorch ==2.1.0
+    - libtorch ==2.1.0
+    - fftw
+    - libtiff
+    - zlib
+    - torchvision
+    - cudnn
```

Pinning to patch versions is a anti-pattern that means you won't pick up any security patches and nearly guarantees that your package will have problems resolving.
```suggestion
  build:
    - {{ compiler('cxx') }}
    - {{ compiler('cuda') }}
    - cmake
    - dotnet ~=6.0
    - pytorch ~=2.1.0
    - libtorch ~=2.1.0
    - fftw
    - libtiff
    - zlib
    - cudnn
    - python
  host:
    - dotnet ~=6.0
    - pytorch ~=2.1.0
    - libtorch ~=2.1.0
    - fftw
    - libtiff
    - zlib
    - torchvision
    - cudnn
  run:
    - dotnet ~=6.0
    - pytorch ~=2.1.0
    - libtorch ~=2.1.0
    - fftw
    - libtiff
    - zlib
    - torchvision
    - cudnn
```
Using https://peps.python.org/pep-0440/#compatible-release